guava/src/com/google/common/escape/ArrayBasedCharEscaper.java

```diff
@@ -65,7 +66,7 @@
    */
   protected ArrayBasedCharEscaper(
       Map<Character, String> replacementMap, char safeMin, char safeMax) {
-
+    
```

Comment:
Why are there still whitespace changes? Please make sure these are gone - we can't merge anything unless it is clean.